src/main/resources/view/simulationMenuView.fxml

```diff
@@ -13,6 +13,5 @@
     <MenuItem text="Live" fx:id="liveItem"/>
     <MenuItem text="Instant finish" fx:id="instantFinishItem"/>
     <MenuItem text="Pause" fx:id="pauseItem"/>
-    <MenuItem text="Simulate" fx:id="simulateItem"/>
-    <MenuItem text="Draw" fx:id="drawItem"/>
+    <MenuItem text="Simulate" fx:id = "simulateItem" onAction="#changeApplicationState"/>
```

Comment:
How about we rename it to "Switch mode"? I'm not convinced that it's intuitive to switch from simulation to draw mode by clicking on "Simulate" item. Or maybe we could change the background colour to mark that simulation mode is active?

Reply:
For now this menu item is changed to toggle button. We can change it in the future, but until we don't have clear vision for it, let's leave it as it is.